packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -866,6 +868,9 @@ class Scaffold extends StatefulWidget {
   /// [AppBar.primary], is true.
   final bool primary;
 
+  /// {@macro flutter.gestures.recognizer.dragStartBehavior}
+  final DragStartBehavior dragStartBehavior;
```

Comment:
this might want to be called drawerDragStartBehavior or some such, and the docs should definitely explain what it's used for.

Reply:
So should I name all the drag behaviors be named specifically according to their functionality?

Reply:
you'd have to determine this on a case-by-case basis. For Scaffold, there's a lot going on in the widget, but the dragStartBehavior only controls what happens with the drawers (right?), so "dragStartBehavior" is overly broad as a name. For GestureDetector, it's much more general and applies to all the gesture recognizers that care, so "dragStartBehavior" is a perfectly reasonable name.